### ISSUE TITLE
Moving Filters pane to the right side

### ISSUE THREAD

**Issue by birbilis** [seajax/seajax#17]
I'm trying to move the Filters pane to the right side (to show under the details pane), but if I use
> right:5px;

instead of
> left:5px;

at .pivot_filterpane in the CSS, then the result items don't show up at the center anymore

**Comment by sethbrenith** [comment]
Yeah, unfortunately this would take more than just a CSS change; there is some script logic that does the centering and I think it is currently assuming that the filters pane takes up a certain amount of the left side. We would have to add some additional configuration when initializing the viewer to explain to it how to center things.
